mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -303,14 +341,15 @@ object GeneralizedLinearRegression extends DefaultParamsReadable[GeneralizedLine
 
   /** Set of family and link pairs that GeneralizedLinearRegression supports. */
   private[regression] lazy val supportedFamilyAndLinkPairs = Set(
-    Gaussian -> Identity, Gaussian -> Log, Gaussian -> Inverse,
-    Binomial -> Logit, Binomial -> Probit, Binomial -> CLogLog,
-    Poisson -> Log, Poisson -> Identity, Poisson -> Sqrt,
-    Gamma -> Inverse, Gamma -> Identity, Gamma -> Log
+    "gaussian" -> Identity, "gaussian" -> Log, "gaussian" -> Inverse,
```

Comment:
@yanboliang  The member object (in `GeneralizedLinearRegressionBase`) won't be accessible in `Family`, right? The method `Family.fromName($(family))` uses global objects like `Poisson`, `Gamma` etc. To use  `Family.fromName`, I need to create a `Tweedie` global object. Then we are back to the issue that @srowen pointed out of setting `variancePower` of the global object. Please advise. Thanks.